src/style/app-fixed.module.css

```diff
@@ -4074,14 +4204,57 @@ form {
 
 /* -- OrgUpdate.tsx -- */
 
-.userupdatediv {
-  border: 1px solid var(--userupdatediv-border);
-  box-shadow: 2px 1px var(--userupdatediv-box-shadow);
-  padding: 25px 16px;
-  border-radius: 5px;
-  background: var(--userupdatediv-bg);
+.orgCardSettings {
+  width: 558px;
 }
 
+.resetChangesBtn {
+  border-width: 2px;
+  border-color: #1778f2;
+  color: #1778f2;
+  padding: 0.5rem 1rem;
+  border-radius: 0.5rem;
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  height: 49px;
+  width: 160px;
+  font-size: 0.8rem;
+}
+
+.resetChangesBtn:hover {
+  color: #1778f2;
+  box-shadow: 2.5px 2.5px 2.5px var(--dropdown-shadow);
+  border: 2px solid #1778f2 !important;
+}
+
+.syncIconStyle,
+.syncIconStyle:focus {
+  transform: rotate(135deg) scale(1.2);
+  width: 10px;
+  height: 10px;
+  stroke: #1778f2;
+  stroke-width: 1;
+  fill: #1778f2;
```

Comment:
Create a variable for this color and purpose. There are examples at the top of the file.